week 5/week-five-task-solution/Roshambo/script.js

```diff
@@ -0,0 +1,117 @@
+const choices = document.querySelectorAll('.choice')
+const score = document.querySelector('#score');
+const result = document.querySelector('#result');
+const restart = document.querySelector('#restart');
+const modal = document.querySelector('.modal');
+const scoreboard = {
+    player: 0,
+    computer: 0
+};
+
+// Play game 
+function play(e) {
+    restart.style.display = 'inline-block';
+    const playerChoice = (e.target.id);
+    const computerChoice = getCompuerChoice();
+    const winner = getWinner(playerChoice, computerChoice);
+    showWinner(winner, computerChoice);
+}
+
+// Get Computer Choice 
+function getCompuerChoice() {
+    const randNum = Math.random();
+
+    if (randNum < 0.35) {
+        return 'rock';
+    } else if (randNum <= 0.67 ) {
+        return 'paper';
+    } else {
+        return 'scissors';
+    }
+}
```

Comment:
You can try attempting turning this into a class

Reply:
Okay
I’ll try do that 

Thanks